hack/config.sh

```diff
@@ -1,8 +1,18 @@
-binaries="cmd/virt-controller cmd/virt-launcher cmd/virt-handler cmd/virt-api cmd/virtctl cmd/virt-manifest"
-docker_images="$binaries images/haproxy images/iscsi-demo-target-tgtd images/vm-killer images/libvirt-kubevirt images/spice-proxy cmd/virt-migrator"
-docker_prefix=kubevirt
-docker_tag=${DOCKER_TAG:-latest}
-manifest_templates="`ls manifests/*.in`"
-master_ip=192.168.200.2
-master_port=8184
-network_provider=weave
+unset binaries docker_images docker_prefix docker_tag manifest_templates \
```

Comment:
Hmm, its only by luck that vagrant isn't already broken - even the existing file used shell expansions for several variables that vagrant is just reading as literal text :-(

I'll try to do a  hack/config-default.cfg & hack/config-local.cfg which are always plain text based, and then use a config.sh wrapper.